### PR TITLE
RA2 - Added Grizzly Tank

### PR DESCRIPTION
**Pull request by MustaphaTR**
Some vaules may be false and i'm not sure is it works because i made it from my tablet and not tried yet.

~~Needs a squash first.~~

### CODE REVIEW

**Comment by Phrohdoh** [comment]
Please refer to the roadmap in the readme.

**Comment by MustaphaTR** [comment]
I know they are in 3. place but as i said i made it from tablet and planned make Soviet Radar first but i dont know its anim frames and i was unable to find it without PC.

**Comment by Phrohdoh** [comment]
That is fine, but neither this or the Soviet Radar will be merged out of order.

**Comment by abcdefg30** [comment]
Needs a rebase.

File: rules/vehicles.yaml [hunk 1]
```diff
@@ -72,3 +72,40 @@ SMCV:
...
+	Turreted:
+		ROT: 5
+	Armament@PRIMARY:
+		Weapon: 105mm
+		LocalOffset: 640,0,384
+		MuzzleSequence: muzzle
+	AttackTurreted:
+	AutoTarget:
+	RenderSprites:
+		Image: GTNK
```
Comment (abcdefg30) [inline rules/vehicles.yaml]: This can be lowercase: `gtnk`.

File: sequences/voxels.yaml [hunk 2]
```diff
@@ -16,11 +16,10 @@ apc: # TODO apcw in water
 	idle:
 
 art2:
-	idle:
-
-gaarty:
-	turret: art2tur
-	barrel: art2barl
+	idle
```
Comment (abcdefg30) [inline sequences/voxels.yaml]: This change doesn't look right.

Comment (MustaphaTR) [inline sequences/voxels.yaml]: They are left from ts. Do we need them? I just replaced it instead of adding new. TS entries should be all deleted later.

**Comment by MustaphaTR** [comment]
Can anybody help me to rebase it. I don't know how to do it. It hapened o me one time before and i opened new request at that time.

**Comment by abcdefg30** [comment]
You could try https://help.github.com/articles/using-git-rebase/ (but a backup before may be good).
Also it'd be good if you join us on IRC.

**Comment by Phrohdoh** [comment]
You can also join us on #orcaware on irc.freenode.net.

**Pull request closed without merging**